### PR TITLE
Add example for using parameters and variables in Script resource

### PR DESCRIPTION
**Pull request by nyanhp**
# PR Summary
Added more concise example for using parameters in Script resource in addition to the existing example 2.

## PR Context

Select the type(s) of documents being changed.

**Cmdlet reference & about_ topics**
- [ ] Version 7.1 preview content
- [ ] Version 7.0 content
- [ ] Version 6 content
- [ ] Version 5.1 content

**Conceptual content**
- [ ] Fundamental conceptual articles
- [ ] Script sample articles
- [x] DSC articles
- [ ] Gallery articles
- [ ] JEA articles
- [ ] WMF articles
- [ ] SDK articles
- [ ] Community content

## PR Checklist

- [x] I have read the [contributors guide][contrib] and followed the style and process guidelines
- [x] PR has a meaningful title
- [x] PR is targeted at the _staging_ branch
- [x] All relevant versions updated
- [ ] Includes content related to issues and PRs - see [Closing issues using keywords][key].
- [x] This PR is ready to merge and is not **Work in Progress**
  - If the PR is work in progress, please add the prefix `WIP:` or `[WIP]` to the beginning of the
    title and remove the prefix when the PR is ready.

[contrib]: https://docs.microsoft.com/powershell/scripting/community/contributing/overview
[key]: https://help.github.com/en/articles/closing-issues-using-keywords

### CODE REVIEW

File: reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md [hunk 1]
```diff
@@ -150,4 +150,67 @@ Configuration ScriptTest
         }
     }
 }
+```
+
+### Example 3: Utilizing parameters in a Script resource
+
+This example accesses parameters from within the Script resource by making use of the using-scope.
```
Comment (chasewilson) [inline reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md:157]:
```suggestion
This example accesses parameters from within the Script resource by making use of the `using` scope.
```

File: reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md [hunk 2]
```diff
@@ -150,4 +150,67 @@ Configuration ScriptTest
...
+Please note that ConfigurationData can be accessed in a similar way. Like example 2, a version is
```
Comment (chasewilson) [inline reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md:158]:
```suggestion
Please note that **ConfigurationData** can be accessed in a similar way. Like example 2, a version is
```

File: reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md [hunk 3]
```diff
@@ -150,4 +150,67 @@ Configuration ScriptTest
...
+            }
+            SetScript = {
+                Set-Content -Path $using:FilePath -Value $using:Version
+            }
+        }
+    }
+}
+```
+
+The resulting MOF file includes the variables and their values accessed through the using-scope.
```
Comment (chasewilson) [inline reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md:205]:
```suggestion
The resulting MOF file includes the variables and their values accessed through the `using` scope.
```

File: reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md [hunk 4]
```diff
@@ -150,4 +150,67 @@ Configuration ScriptTest
...
+They are injected into each scriptblock which uses the variables
+(Test and Set scripts removed for brevity):
```
Comment (chasewilson) [inline reference/docs-conceptual/dsc/reference/resources/windows/scriptResource.md:207]: You could remove the parenthesis here.

**Comment by nyanhp** [comment]
@chasewilson thank you very much for your feedback :) I incorporated the suggestions

**Comment by chasewilson** [comment]
@nyanhp Thanks again for the contribution and everything looks good here. You should see it post to the docs page within 24 after the PR is merged. 

Have a great day!

**Pull request merged**